VEHICLE/N_0x8aa9180de2fedd45.md

```diff
@@ -1,15 +1,19 @@
 ---
 ns: VEHICLE
+aliases: ["_0x8AA9180DE2FEDD45"]
 ---
-## _0x8AA9180DE2FEDD45
+## EnableAircraftObstacleAvoidance
```

Comment:
This should be uppercase, I sent you an example file in a previous comment, but I'll send it again just in case 😅

i.e. `_ENABLE_AIRCRAFT_OBSTACLE_AVOIDANCE`

[EnableAircraftObstacleAvoidance(1).md](https://github.com/citizenfx/natives/files/11908160/EnableAircraftObstacleAvoidance.1.md)

Reply:
Geez man! I'm so sorry! I thought I made the appropriate changes! I'll get it eventually! LOL

Anyways, hopefully this will work, but if anything's missing, please let me know! I promise I'm learning! 🤣